.github/workflows/refresh_team_page.yml

```diff
@@ -1,15 +0,0 @@
-name: Refresh Team Page
-
-on:
-  push:
-    branches: [ main ]
-
-permissions: {}
-jobs:
-  build:
-    name: Refresh Contributors Stats
-    runs-on: ubuntu-latest
-    steps:
-      - name: Call GitHub API
-        run: |
-          curl "https://api.github.com/repos/${{ github.repository }}/stats/contributors"
```

Comment:
Do we really need this change here?

Reply:
Yeah you're right, I let my hand slip. This should go in another PR.
Will remove now 🙂